php/cloudbuild-release.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# note: all local files are placed under /workspace and are required in path
+
+timeout: 21600s  # 6 hours
+options:
+ machineType: 'E2_HIGHCPU_8'
+ logging: CLOUD_LOGGING_ONLY
+steps:
+
+# PHP 8.1
+- id: 'build-php81'
```

Comment:
I'm not sure what the context is here, but since PHP 8.1 will be EOL in November, maybe we use PHP 8.3?

Reply:
thanks @bshaffer for pointing out.  It looks like currently the php team is using Php 8.1 for their [release jobs](https://source.corp.google.com/piper///depot/google3/cloud/sdk/client_libraries/kokoro/php/googleapis/google-api-php-client-services/release.cfg;l=17).  Should that be migrated to use php 8.3?  do all php client lib projects support 8.3?

Reply:
Let's migrate as-is and when PHP want's to migrate, we can do the migration separately.